obspy/io/sac/LICENSE.txt

```diff
@@ -0,0 +1,16 @@
+Copyright 2015. Los Alamos National Security, LLC for pysac LA-CC-15-051. This
```

Comment:
I had to call it something when it was processed for release.  I'm fuzzy on what my obligations are, but I bet they'd be satisfied if I put everything new under `obspy/io/sac/pysac` (including the wonky redundant LICENSE.txt) so that it nominally retains that name.  Thoughts?